sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -288,6 +297,65 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with SharedSQLContext
     }
   }
 
+  test("SQL interface support storageLevel(DISK_ONLY)") {
+    sql("CACHE TABLE testData OPTIONS('storageLevel' 'DISK_ONLY')")
+    assertCached(spark.table("testData"))
+    val rddId = rddIdOf("testData")
+    assert(isExpectStorageLevel(rddId, Disk))
+    spark.catalog.uncacheTable("testData")
```

Comment:
Let's remove `uncacheTable` since we have `spark.catalog.clearCache()` in `afterEach()`.